chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md

```diff
@@ -45,9 +45,10 @@ Multiple licenses can be represented using a SPDX license expression as defined
 
 Examples:
 
-    SPDX-License-Identifier: (GPL-2.0 OR MIT)
-    SPDX-License-Identifier: (LGPL-2.1 AND BSD-2-CLAUSE)
-    SPDX-License-Identifier: (GPL-2.0+ WITH Bison-exception-2.2)
+    SPDX-License-Identifier: (GPL-2.0-only OR MIT)
+    SPDX-License-Identifier: (LGPL-2.1-only AND BSD-2-CLAUSE)
```

Comment:
Both here and in the original, it looks like the "BSD-2-CLAUSE" was incorrectly capitalized. The correct version should be "BSD-2-Clause". 

I think that capitalization might not matter for license identifier syntax, but it would be great if we can match the capitalization that's on the license list for consistency's sake.

Reply:
Agreed.